src/append/rolling_file/rolling.rs

```diff
@@ -280,9 +275,8 @@ impl State {
                     }
                 }
 
-                if self.log_filename_prefix.is_none()
-                    && self.log_filename_suffix.is_none()
-                    && Date::parse(filename, &self.date_format).is_err()
+                if self.log_filename_prefix.is_none() && self.log_filename_suffix.is_none()
+                // && Date::parse(filename, &self.date_format).is_err()
```

Comment:
@BurntSushi this original logic requires a counterpart of `strftime`.

That is, parse filename in:

```
        match *self {
            Rotation::Minutely => "%F-%H-%M",
            Rotation::Hourly => "%F-%H",
            Rotation::Daily => "%F",
            Rotation::Never => "%F",
        }
```

accordingly.

`DateTimeParser` doesn't seems to be able to customize the format.

Reply:
Yeah it's not intended for `jiff::fmt::temporal` to provide much customization because it's conforming to an external specification. Using `strftime` for a custom format like this is one of its intended use cases.

Reply:
Found `strptime` the counterpart of `strftime`.

Reply:
Oh I didn't understand your question. Yes, `strptime` is for parsing, `strftime` is for formatting.